src/main/java/ConsoleInput.java

```diff
@@ -0,0 +1,91 @@
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.regex.Pattern;
+
+public class ConsoleInput {
+
+    public ConsoleInput() {
```

Comment:
What is this chap here doing?

Reply:
Maybe you could use this for greater things.

Reply:
If this looked something like this, it would be great:

```
public ConsoleInput (InputStream input, OutputSteam output) {
      ///assign input and output to instance variables
}
```